RAGens/common/src/G_main.cpp

```diff
@@ -552,8 +552,6 @@ int Set_Render(HWND hWnd, int Full, int Num, BOOL fForceWindowReposition)
 	Build_Main_Menu();
 	int nResult = Init_DDraw(HWnd);
 	
-	RA_InitDirectX();
-
```

Comment:
function body was commented out in RA_Integration. exposed method has been removed.